src/transaction/sign.ts

```diff
@@ -95,7 +95,7 @@ function objectDiff(a: object, b: object): object {
       return
     }
     if (type1 === '[object Array]') {
-      if (!isEqual(i1, i2)) {
+      if (!_.isEqual(i1, i2)) {
```

Comment:
Why is it important to change the imports here?

Reply:
`lodash.isequal` is technically a separate package, which takes up separate space. I removed the package as a dependency.